react/OrderByFlexible.js

```diff
@@ -5,15 +5,15 @@ import OrderBy from './OrderBy'
 
 import styles from './searchResult.css'
 
-const withSearchPageContextProps = Component => () => {
+const withSearchPageContextProps = Component => (props) => {
   const { orderBy } = useSearchPage()
   if (orderBy == null) {
     return null
   }
 
   return (
     <div className={styles['orderBy--layout']}>
-      <Component orderBy={orderBy} />
+      <Component {...props} orderBy={orderBy} />
```

Comment:
what is the purpose of this change?

Reply:
by that I mean, can you just pass down the specific prop you need?

Reply:
You mean this?

```
const withSearchPageContextProps = Component => ({hiddenOptions}) => {
 // omitted
<Component hiddenOptions={hiddenOptions} orderBy={orderBy} />
// omitted
}
```